3-morse-code-translator/practice.py

```diff
@@ -0,0 +1,37 @@
+def morse_code_translator(message):
+    # empty list
+    result = []
+    # morse code table
+    morse_code_dict = {
+    # alphabet
+    'A': '.-',    'B': '-...',  'C': '-.-.',  'D': '-..',   'E': '.', 
+    'F': '..-.',  'G': '--.',   'H': '....',  'I': '..',    'J': '.---', 
+    'K': '-.-',   'L': '.-..',  'M': '--',    'N': '-.',    'O': '---', 
+    'P': '.--.',  'Q': '--.-',  'R': '.-.',   'S': '...',   'T': '-', 
+    'U': '..-',   'V': '...-',  'W': '.--',   'X': '-..-',  'Y': '-.--', 
+    'Z': '--..',
+
+    # numbers
+    '0': '-----', '1': '.----', '2': '..---', '3': '...--', '4': '....-', 
+    '5': '.....', '6': '-....', '7': '--...', '8': '---..', '9': '----.',
+    
+    # special characters
+    '.': '.-.-.-',  ',': '--..--',  '?': '..--..',  "'": '.----.',  '!': '-.-.--',
+    '/': '-..-.',   '(': '-.--.',   ')': '-.--.-',  '&': '.-...',   ':': '---...',
+    ';': '-.-.-.',  '=': '-...-',   '+': '.-.-.',   '-': '-....-',  '_': '..--.-',
+    '"': '.-..-.',  '$': '...-..-', '@': '.--.-.', ' ': '/'
+}
+    message = message.upper()
+    for index, char in enumerate(message):
+        if char in morse_code_dict.keys():
+            morse_code = morse_code_dict.get(char)
+        else:
+            print(f"Skipping unsupported {char} at position {index + 1}")
+            continue
+        result.append(morse_code)
```

Comment:
```suggestion
        morse_code = morse_code_dict.get(char)
        if morse_code is None:
            result.append(morse_code)
        else:
            print(f"Skipping unsupported {char} at position {index + 1}")
```
Just a small suggestion: since `dict.get()` returns `None` by default if the key doesn't exist, the code could be rewritten as follows for better readability.